File: 0242-valid-anagram/0242-valid-anagram.cpp
class Solution {
public:
    bool isAnagram(string s, string t) {
        map<char,int> sMap; 
        for(char c : s) sMap[c]++;
        for(char c : t) sMap[c]--; 
        for(auto& it : sMap){
            if(it.second != 0) return false; 
        }
        return true;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.java
class Solution {
    public boolean isAnagram(String s, String t) {
        char[] map = new char[26]; 
        for(char c : s.toCharArray()) map[c-'a']++; 
        for(char c : t.toCharArray()) map[c-'a']--; 

        for(int i = 0; i < 26; i++){
            if(map[i] != 0) return false; 
        }
        return true; 
    }
}